app/config/config.yml

```diff
@@ -8,6 +8,7 @@ imports:
 parameters:
     locale: en
     AdapterSecurityAdminClass: PrestaShop\PrestaShop\Adapter\Security\Admin
+    authorization_employee: ['view','add','edit','delete']
```

Comment:
[YAGNI](https://fr.wikipedia.org/wiki/YAGNI) I'd like to see theses authorizations to be part of AuthorizationChecker class.